gui/default/syncthing/core/aboutModalView.html

```diff
@@ -9,6 +9,8 @@ <h1 class="text-center">
     </h1>
     <hr />
 
+    <p><span translate>Syncthing is Free and Open Source Software licensed as</span> <a href="https://github.com/syncthing/syncthing/blob/master/LICENSE">MPL v2.0</a>.</p>
```

Comment:
I don't think we can break a translated sentence like this; other languages might need to put the "MPL 2.0" string somewhere in the middle.

Reply:
Good point.